spring-cloud-sleuth-core/src/main/java/org/springframework/cloud/sleuth/Span.java

```diff
@@ -31,16 +31,28 @@
  * Class for gathering and reporting statistics about a block of execution.
  * <p/>
  * Spans should form a directed acyclic graph structure. It should be possible to keep
- * following the parents of a span until you arrive at a span with no parents.
+ * following the parents of a Span until you arrive at a Span with no parents.
```

Comment:
nit: I don't think capitalization helps.. it is close to `{@link` or `{@linkplain`, but not.. I'd just leave it as it was.